src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-api-gateway-v1-monitoring-integration.mdx

```diff
@@ -1,21 +1,25 @@
 ---
-title: Amazon API Gateway monitoring integration
+title: Amazon API Gateway V1 monitoring integration
 tags:
   - Integrations
   - Amazon integrations
   - AWS integrations list
 metaDescription: "New Relic's AWS API Gateway monitoring integration: what data it reports, and how to enable it."
 redirects:
-  - /docs/integrations/amazon-integrations/aws-integrations-list/aws-api-gateway-monitoring-integration
+  - /docs/integrations/amazon-integrations/aws-integrations-list/aws-api-gateway-v1-monitoring-integration
   - /docs/aws-apigateway-integration
-  - /docs/infrastructure/amazon-integrations/amazon-integrations/aws-api-gateway-monitoring-integration
+  - /docs/infrastructure/amazon-integrations/amazon-integrations/aws-api-gateway-v1-monitoring-integration
```

Comment:
@rahul188 I think we also want to preserve the original redirect since there never was a page with the "v1" in the filename. We want anyone with the old link to be redirected here to this renamed v1 page, right?

Reply:
@rhetoric101, I made the suggested change.